### DISCUSSION TITLE
Garage door problem

### DISCUSSION THREAD

**Discussion by hatrick9** [RavenSystem/esp-homekit-devices#1798] [MEPLHAA Scripts]
After the last update, I have a problem with the garage door.  Every now and then I have "no answer".  Maybe the problem is in the code.  please somebody analyze it.  There was no problem with that.  Before the update, it worked great.
I have: SONOFF SV GATE LOW VOLTAGE 5-24v DC with reed switch.

Poniżej kod
`{"c":{"l":13,"b":[{"g":0,"t":5}]},"a":[{"t":40,"d":15,"f3":[{"g":14,"t":0}],"f4":[{"g":14}],"0":{"r":[{"g":12,"v":1,"i":0.5}]},"1":{"a":0},"2":{"r":[{"g":12,"i":0.5},{"g":12,"v":1,"i":0.5},{"g":12,"v":1,"i":2}]},"3":{"a":2}}]} `

**Comment by ceg117** [comment]
Garage Door  json
{"c":{"o":0},"a":[{"0":{"r":[{"g":4,"v":1},{"g":5,"v":0}]},"1":{"r":[{"g":4,"v":0},{"g":5,"v":1}]},"2":{"r":[{"g":4,"v":1},{"g":5,"v":0}]},"3":{"r":[{"g":4,"v":0},{"g":5,"v":1}]},"4":{"r":[{"g":4,"v":0},{"g":5,"v":0}]},"5":{"r":[{"g":4,"v":0},{"g":5,"v":0}]},"6":{"r":[]},"7":{"r":[]},"8":{"r":[]},"9":{"r":[]},"10":{"r":[{"g":4,"v":0},{"g":5,"v":0}]},"t":40,"s":1,"d":10,"b":[],"f6":[{"g":12,"t":1}],"f7":[{"g":12,"t":0}],"f8":[{"g":12,"t":1},{"g":12,"t":0}],"f1":[{"g":3,"t":0}],"f2":[{"g":13,"t":0}],"f3":[{"g":14,"t":0}],"f4":[],"f0":[{"g":0,"t":0}]}]}

**Comment by ceg117** [comment]
direct current machine
